solutions/usaco-988.mdx

```diff
@@ -2,11 +2,14 @@
 id: usaco-988
 source: USACO Bronze 2020 January
 title: Photoshoot
-author: Ananth Kashyap
+author: Ananth Kashyap, Ben Dodge
 ---
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_photo_bronze_jan20.html)
 
+We'll start by saying that for each $i$ in range $[1, N]$, $a_1 = i$. For all other values of $a_i$, we are given $b_i = a_i + a_{i+1}$ for $[1, N)$, or $a_i = b_{i -1} - a_{i - 1}$ for $[2, N]$. We will then check if these values of $a_i$ produce a valid permutation. 
```

Comment:
why [1, N) and one and [2, N] for the other? probably better to keep consistent, so use (1, N] or [1, N - 1]